src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -592,6 +596,43 @@ public function boot(Application $app)
         }
     }
 
+    /**
+     * @param ContainerInterface $container
+     *
+     * @return void
+     */
+    protected function addRouter(ContainerInterface $container)
+    {
+        if (!$container->has('routers')) {
```

Comment:
Could you put this in a constant?

Reply:
This class will be deprecated soon anyway, using a single const when we have more then hundreds of magic strings doesn't bring any benefit.